docs/sphinx/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = _build
```

Comment:
@utkarsharma2 what are your thoughts on adding `_build` and all its content to `.gitignore`?

Reply:
+1 to this